vtex/billing-app
# Log free installs

#### What is the purpose of this pull request?
Log free installs

#### What problem is this solving?
Free installs were not being logged

#### How should this be manually tested?
1. Clone/pull this repo/branch and link in your workspace
2. Install any app. All installs should be logged into splunk
3. Test this by using this query at Splunk:
`index=colossus key=log_info sender="vtex.billing@0.5.1" account={{yourAccount}} workspace={{yourWorkspace}}

#### Types of changes
